src/pfe/file-watcher/server/src/projects/OdoExtensionProject.ts

```diff
@@ -23,14 +23,40 @@ import { ProjectInfo, BuildLog, AppLog, ProjectCapabilities, defaultProjectCapab
 import { Validator } from "./Validator";
 import { IExtensionProject } from "../extensions/IExtensionProject";
 
+// skeleton for the logs originated from the extension json
```

Comment:
@jingfuwang Could you please review the ODOExtensionProject part of this PR? Thanks.